Document some WinHttp Options and some Query Info Flags

trmanc opened the pull request.

trmanc committed:
Document some WinHttp Options and some Query Info Flags

PRMerger16 commented:
@trmanc : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edits

Just speaking to the developer rather than about them, and removing the passive voice.

PRMerger15 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edits

Removing passive voice. Also we like to avoid saying things like "note that" or "be aware that" or "understand that", and instead just say the thing. If something really needs to stand out then we can add [!NOTE] (or TIP or IMPORTANT).

I've also changed things like "will perform" (which speaks about the future) to just "performs". "Performs" flattens out time and tense, removes any hint that we might be talking about some event or point in time or other condition that we omitted, and instead simply says what the feature does under the conditions we specified. Rather than "what it *will* do", which seems to introduce *some new, uncertain, future factor*.

PRMerger12 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
